src/pages/components/structured-list/usage.mdx

```diff
@@ -138,16 +162,86 @@ alignment.
+has the following states: enabled, hover, focus, selected, disabled, and
+skeleton. For detailed visual information about the various states for this
+component, refer to the [Style](/components/structured-list/style) tab.
+
+| State      | When to use                                                                                                                                                                                                                       |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Enabled_  | When an action in the list is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component.                                                                  |
+| _Hover_    | When a user’s mouse cursor is hovering over the structured list item.                                                                                                                                                             |
+| _Focus_    | When a user tabs or clicks on a row, it becomes focused, indicating the user has successfully navigated to the component (except default structured list).                                                                        |
+| _Selected_ | When a user presses space or clicks on a structured list row to select it.                                                                                                                                                        |
+| _Disabled_ | When the user is not allowed to interact with a selectable structured list due to either permissions, dependencies, or pre-requisites.                                                                                            |
+| _Skeleton_ | When the user can review but not modify the component. This state removes all interactive functions like the disabled state but can still be focusable, accessible by screen readers, and passes visual contrast for readability. |
+
 ## Interactions
 
 #### Selectable structured list
 
 - Only one item can be selected from the list.
-- By default, one option should be selected.
+- By default, no option should be selected.
 - If you need to select multiple items, use a
   [data table.](/components/data-table/usage)
 - When the user selects an item from the list, the selected row will appear with
-  the `checkmark--filled` icon.
+  the `radio-button--checked` icon.
+
+## Default
+
+Default structured lists are used to display information to the user, such as
+pricing plans. These structured list can have internal calls-to-action (CTAs),
+such as a [button](/components/button/usage) or
+a [link](/components/link/usage).
+
+<Row>
+<Column colLg={12}>
+
+![default structured list](images/structured-list-usage-4.png)
```

Comment:
Just some typos:

- In the left most column, the word "Related" has a typo.
- There is an extra space in the first paragraph between "Director" and "platform".
- There is an extra space in the first paragraph between "hosting," and "operations,".
- There is an extra space in the first paragraph between "can" and "quickly".
- Under the "Single-tenant" column in the structured list, the first sentence should say the word "require" not the word required.
- Under the "Multi-tenant" column in the structured list,  in the second row, the beginning of the sentence needs to be capitalized and be "On-demand".